.github/workflows/style.yml

```diff
@@ -41,6 +41,12 @@ jobs:
         with:
           distribution: zulu
           java-version: 8
+      - name: install kyuubi with the latest commit
```

Comment:
RELATED?

Reply:
the new added failed will fail in style check workflow if we do not install first. So I changed the ga in this PR